### PR TITLE
Add functionality for displaying routes based on logged in driver

### PR DESCRIPTION
**Pull request by sahilpatel1906**

### CODE REVIEW

**Review by Tommybui28 — APPROVED** [review]
Very good!

**Review by ney601 — APPROVED** [review]

**Review by Jean-005 — APPROVED** [review]
awesome!

File: capstone_frontend/src/containers/DeliveryContainer.jsx
Comment (Aebel-Shajan) [inline capstone_frontend/src/containers/DeliveryContainer.jsx:1]: Replace loading routes with "please log in"

File: capstone_frontend/src/components/RouteDisplay.jsx [hunk 1]
```diff
@@ -14,10 +14,13 @@ const maps = {
 const RouteDisplay = ({route}) => {
   const [map, setMap] = useState(null);
 
-  if(route.waypoints.length === 0) {
+  if(route.length === 0) {
```
Comment (Aebel-Shajan) [inline capstone_frontend/src/components/RouteDisplay.jsx:17]: Nice check

**Review by Aebel-Shajan — APPROVED** [review]
lgtm

**Pull request merged**